webui/react/src/utils/workload.ts

```diff
@@ -1,12 +1,12 @@
-import { CheckpointState, CheckpointWorkload, MetricsWorkload, Step,
+import { CheckpointWorkload, MetricsWorkload, Step,
   WorkloadWrapper } from '../types';
 
 export const hasCheckpointStep = (step: Step): boolean => {
-  return !!step.checkpoint && step.checkpoint.state !== CheckpointState.Deleted;
+  return !!step.checkpoint;
```

Comment:
**question:** help me understand this part. Is it that we no longer support checkpoint deletion or should checkpoints be shown even if deleted?

Reply:
I chatted with @stoksc  and as a result, I brought back the checkpoint state.